### DISCUSSION TITLE
project

### DISCUSSION THREAD

**Discussion by Anurag9981** [Praful932/Kitabe#264] [General]
https://user-images.githubusercontent.com/85742953/167278775-15a1c135-6045-4b72-8eee-763f3180d079.mp4
Hey sir,
 there is a problem in this project, i am searching the book so i am doing the recommendation but if i am clicking on the recommendation then the show is not happening
ye android me work kar raha h par pc me nhi  plz help me sir😊

**Comment by Praful932** [comment]
@Anurag9981 have added comments in issue #265

**Comment by ranjithloda** [comment]
Can anybody say how to execute this project
